WordPress/src/test/java/org/wordpress/android/ui/activitylog/list/filter/ActivityLogTypeFilterViewModelTest.kt

```diff
@@ -172,15 +181,27 @@ class ActivityLogTypeFilterViewModelTest : BaseUnitTest() {
     @Test
     fun `items are checked, when the user opens the screen with active activity type filter`() = test {
         val uiStates = init().uiStates
-        val initialSelection = listOf(1, 4)
+        val initialSelection = listOf("1", "4")
 
         startVM(initialSelection = initialSelection)
 
         assertThat((uiStates.last() as Content).items.filterIsInstance(ActivityType::class.java)
-                        .filter { it.checked }.map { it.id }
+                .filter { it.checked }.map { it.id }
         ).containsExactlyElementsOf(initialSelection)
     }
 
+    @Test
+    fun `date range passed to the api, when provided on view model start`() = test {
+        init().uiStates
+        val captor: KArgumentCaptor<FetchActivityTypesPayload> = argumentCaptor()
+
+        startVM(dateRange = Pair(1L, 2L))
```

Comment:
Minnor (🔍): Consider using a variable instead of hardcoded duplicate values for when creating the action and asserting afterwards that the parameter put to the actions is the one expected.
```
val after = 1L
val before = 2L

startVM(dateRange = Pair(after, before))

assertThat(captor.firstValue.after?.time).isEqualTo(after)
assertThat(captor.firstValue.before?.time).isEqualTo(before)
```

PS: This way the test because more concrete. Wdyt?

Reply:
:D Funny, I do this in 95% cases. I considered it even in this method, but it felt like it's not worth it. Since you pointed it out, it's probably worth it. Fixed in 28163726184bf8f142f1108ca6fff8f41fdf9764